pytorch_lightning/core/lightning.py

```diff
@@ -280,6 +280,7 @@ def log(
         add_dataloader_idx: bool = True,
         batch_size: Optional[int] = None,
         metric_prefix_name: Optional[str] = None,
+        is_global_zero: Optional[bool] = None,
```

Comment:
can't we infer this by checking the rank? this shouldn't be exposed to the user IMO

Reply:
Not sure how we can. I tried by checking modify the TPCStore directly, but didn't work as expected.